.github/workflows/ci.yml

```diff
@@ -52,6 +52,8 @@ jobs:
         with:
           command: deny
           args: check Licenses # check that all of deps have an ok license.
+      - name: install cmake
```

Comment:
why do we need cmake here?

Reply:
`cargo doc` is failed on runner, and the message indicate that there is no cmake which is needed for crate `rdkafka-sys`. So I added a step to install cmake.

https://github.com/quickwit-oss/quickwit/runs/5531433392?check_suite_focus=true#step:8:158

Do you have a better idea?